lib/popup.js

```diff
@@ -3,104 +3,90 @@ const measureDisplayEl = document.querySelector('.measure-display')
 const timeSpentEl = document.querySelector('.time-spent')
 const networkEl = document.querySelector('.network')
 const dateEl = document.querySelector('.date')
-let displayDate = 'Today'
-let displayHostname = ''
+const trackButton = document.getElementById('trackButton')
 
 const dates = {
   'Today': new Date(new Date().setHours(0, 0, 0, 0)),
   'Yesterday': new Date(new Date(new Date().setDate(new Date().getDate() - 1)).setHours(0, 0, 0, 0))
 }
 
-for (const date in dates) {
-  const dateItem = document.createElement('option')
-  dateItem.textContent = date
-  dateItem.value = date
-  dateEl.appendChild(dateItem)
-}
-
-chrome.storage.local.get({ hostNavigationStats: {}, config: {} }, results => {
-  // Return if the stats object is empty
-  if (Object.keys(results.hostNavigationStats).length === 0) {
-    return
-  }
-
-  const todayStats = results.hostNavigationStats[dates['Today']]
-  const sortedHostnames = Object.keys(todayStats).sort((a, b) => {
-    return todayStats[a] <= todayStats[b]
+chrome.storage.local.get({ hostNavigationStats: {}, config: {} }, res => {
+  chrome.tabs.query({active: true, currentWindow: true}, tabs => {
+    view(new URL(tabs[0].url).hostname, 'Today', res.config, res.hostNavigationStats)
   })
+})
 
-  if (sortedHostnames.length === 0) {
-    return
-  }
+function view (hostname, date, config, stats) {
+  const dayStats = stats[dates[date]]
+  const sortedHostnames = Object.keys(dayStats).sort((a, b) => dayStats[a] <= dayStats[b]).slice(0, 5)
```

Comment:
Please add a little comment to explain that you sort the host names and only display the five first ones.